src/stream/src/executor/aggregation/approx_count_distinct.rs

```diff
@@ -384,6 +384,26 @@ mod tests {
         test_register_bucket_get_and_update_inner::<4>();
     }
 
+    #[test]
+    fn test_error_ratio() {
```

Comment:
Great test. Can we add a comment about the probability that error_ratio < 0.01? Just to make sure we don't get a flakey test.

Reply:
Also, if the `get_max` function is wrong, does this test fail? In other words, does this test act as a regression test for the get_max bug?

Reply:
> Also, if the `get_max` function is wrong, does this test fail? In other words, does this test act as a regression test for the get_max bug?

Yes, if we don't fix the `get_max` function, this test will fail.